package.json

```diff
@@ -2,10 +2,6 @@
   "name": "frontend",
   "version": "0.1.0",
   "private": true,
-  "workspaces": [
-    "dev/eslint-plugin-guardian-frontend",
-    "tools/amp-validation"
```

Comment:
This will mean dependencies for the `amp-validation` tool will no longer be installed when we run `yarn` at the root level. Will this break any assumptions in CI, or in some cron job somewhere that runs amp validation every x minutes?

Reply:
according to the makefile, we install them whenever we run it?

https://github.com/guardian/frontend/blob/919ef8dc13007f64b14e7d45b3d24aed57a0ccef/makefile#L119-L120

do you know if it's run any other way?

Reply:
Ah okay, that looks legit. I'm not sure if it's run in another way somewhere. I guess we'll soon find out if it is, and we'll know why it's failing! 😄

Reply:
The periodic TC job is here: `teamcity url`/buildConfiguration/dotcom_AmpValidation/578037?buildTab=log&focusLine=3&linesState=587.659

Reply:
it run `npm install` from `tools/amp-validation` so that looks ok. but it hasn't passed for as long as i can see records for, and runs in node 6 as well!

might need a bit of love

Reply:
ha, yes, it's one of those 'would be useful with a couple hours work' things...

Reply:
"when we have spare couple hours we can sort out the backlog of things to do when we have a spare couple of hours"